platforms-serverless/serverless-framework-lambda/serverless.yml

```diff
@@ -19,7 +19,8 @@ functions:
 # only include the Prisma binary required on AWS Lambda while packaging
 package:
   patterns:
-    - '!node_modules/.prisma/client/query-engine-*'
-    - '!node_modules/.prisma/client/libquery_engine-*'
-    - 'node_modules/.prisma/client/libquery_engine-rhel-openssl-1.0.x.so.node'
-    - 'node_modules/.prisma/client/query-engine-rhel-openssl-1.0.x'
+    - '!node_modules/**'
+    - '!prisma/client/query-engine-*'
+    - '!prisma/client/libquery_engine-*'
+    - 'prisma/client/libquery_engine-rhel-openssl-1.0.x.so.node'
+    - 'prisma/client/query-engine-rhel-openssl-1.0.x'
```

Comment:
I think this is from a best practice article we have somewhere.
If we change this here, we should also update the article in docs.

Reply:
https://github.com/prisma/ecosystem-tests/pull/3463#discussion_r1163418117